runtime/src/bank.rs

```diff
@@ -6784,6 +6784,10 @@ impl TransactionProcessingCallback for Bank {
         self.feature_set.clone()
     }
 
+    fn get_epoch_vote_accounts(&self, epoch: Epoch) -> Option<&VoteAccountsHashMap> {
```

Comment:
Why this alias?

Reply:
I can do that, sure!

Reply:
Actually, this does require downstream crates to need the `svm` dependency in order to use this method. I think it actually does make sense to alias it here.